challenge5.js

```diff
@@ -0,0 +1,92 @@
+var billValuesJohn= [124, 48, 268, 180, 42];
+var billValuesMark= [77, 375, 110, 45];
+
+
+calcTipJohn = function(sum){
+    if (sum > 0 && sum < 50){
+        return sum*0.2;
+    } else if (sum => 50 && sum < 200) {
+        return sum*0.15;   
+    } else if (sum >= 200) {
+        return sum*0.1;
+    } else {
+        return 0;
+    }
+}
+
+calcTotalJohn= function(sum){
+    if (sum > 0 && sum < 50){
+        return sum*1.2;
+    } else if (sum => 50 && sum < 200) {
+        return sum*1.15;   
+    } else if (sum >= 200) {
+        return sum*1.1;
+    } else {
+        return 0;
+    }
+}
+
+calcTipMark = function(sum){
+    if (sum > 0 && sum < 100){
+        return sum*0.2;
+    } else if (sum => 100 && sum < 300) {
+        return sum*0.10;   
+    } else if (sum >= 300) {
+        return sum*0.25;
+    } else {
+        return 0;
+    }
+}
+
+calcTotalMark= function(sum){
+    if (sum > 0 && sum < 100){
+        return sum*1.2;
+    } else if (sum => 100 && sum < 300) {
+        return sum*1.10;   
+    } else if (sum >= 300) {
+        return sum*1.25;
+    } else {
+        return 0;
+    }
+}
+
+allTipsJohn = [];
+allTotalsJohn= [];
+allTipsMark = [];
+allTotalsMark= [];
+
+
+for (i in billValuesJohn){
```

Comment:
this is fine, but a bit "old school way", you would get more style points using `map`